src/tunnel.c

```diff
@@ -145,6 +145,14 @@ static int pppd_run(struct tunnel *tunnel)
 			NULL // terminal null pointer required by execvp()
 		};
 
+		if (tunnel->config->pppd_call != NULL) {
```

Comment:
Just write:
 ```
		if (tunnel->config->pppd_call) {
 ```
Again I do not care myself but but the [checkpatch.pl](https://github.com/torvalds/linux/blob/master/scripts/checkpatch.pl) script of the Linux kernel emits a warning and we try to follow the Linux kernel coding style.